README.md

```diff
@@ -107,8 +107,27 @@ Note: specifying `null` in JSON (or equivalent values in other formats) as value
 ## GET
 
 For HTTP GET requests, the query parameters should be provided in the query component of the request URL in the form of
-`key=value` pairs with `&` symbol as a separator.
-The value of the `variables` parameter should be represented as a JSON-encoded string.
+`key=value` pairs with `&` symbol as a separator and both the key and value having their "reserved" characters percent-encoded as specified in [section 2 of RFC3986](https://tools.ietf.org/html/rfc3986#section-2).
+The unencoded value of the `variables` parameter should be represented as a JSON-encoded string.
+
+For example, a function in JavaScript to build the URL for an operation might look like:
```

Comment:
Not sure we should add JS code into spec.
I think the bellow example should be enough.

Reply:
Is not an example implementation useful / in-scope?

Reply:
(Happy to remove.)

Reply:
I see value in having code examples like this, although we might have them as part of a conformance test suite/reference implementation.